src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -119,66 +118,65 @@ function FilterBarUI(props: Props) {
   }
 
   function onAdd(filter: Filter) {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:added`);
     setIsAddFilterPopoverOpen(false);
-    if (props.trackUiMetric) {
-      props.trackUiMetric(METRIC_TYPE.CLICK, `${props.appName}:filter_added`);
-    }
+
     const filters = [...props.filters, filter];
     onFiltersUpdated(filters);
   }
 
   function onRemove(i: number) {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:removed`);
     const filters = [...props.filters];
     filters.splice(i, 1);
     onFiltersUpdated(filters);
   }
 
   function onUpdate(i: number, filter: Filter) {
-    if (props.trackUiMetric) {
-      props.trackUiMetric(METRIC_TYPE.CLICK, `${props.appName}:filter_edited`);
-    }
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filte:edited`);
     const filters = [...props.filters];
     filters[i] = filter;
     onFiltersUpdated(filters);
   }
 
   function onEnableAll() {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:enable_all`);
     const filters = props.filters.map(enableFilter);
     onFiltersUpdated(filters);
   }
 
   function onDisableAll() {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:disable_all`);
     const filters = props.filters.map(disableFilter);
     onFiltersUpdated(filters);
   }
 
   function onPinAll() {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:pin_all`);
     const filters = props.filters.map(pinFilter);
     onFiltersUpdated(filters);
   }
 
   function onUnpinAll() {
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:unpin_all`);
     const filters = props.filters.map(unpinFilter);
     onFiltersUpdated(filters);
   }
 
   function onToggleAllNegated() {
-    if (props.trackUiMetric) {
-      props.trackUiMetric(METRIC_TYPE.CLICK, `${props.appName}:filter_invertInclusion`);
-    }
+    reportUiCounter?.(METRIC_TYPE.CLICK, `filter:invert_all`);
```

Comment:
Does this change how we report existing metrics? Isn't it breaking as it would make those harder to analyze?

Reply:
We never analyze them anyway...
I thought it was good timing for some standardization

Reply:
Since this was added by `kibana/app` https://github.com/elastic/kibana/pull/82709, https://github.com/elastic/kibana/pull/82961
I think we should ping them for reviewing these changes? In case they analyzing those.

Reply:
@majagrubic would you mind taking a look?
I wanted to align the names of telemetry events. Will this break anything for you?

Thanks!

Reply:
Is my understanding correct here that we're no longer tracking the `appName` with the metric? This would make it impossible for us to track which clicks came from inside Discover for example?

Reply:
As I understood, we get an `appname` thanks to the binding right here:
https://github.com/elastic/kibana/pull/91428/files#diff-b370985f7060e1ee97121c246b0ac878bf5b1dee2271e0593a983f893141109eR47

Reply:
@majagrubic We still report the `appName`, I just renamed some of the events themselves

Reply:
I think it's fine with us, we're not actively using these metrics yet.